src/app/header/header.component.html

```diff
@@ -189,8 +191,16 @@ <h5 id="offcanvasStartLabel">penda</h5>
                         />
                         <span class="dark-mode-slider"></span>
                     </label>
-                </div>
+                </div>        
             </div>
+            <ng-template 
+            [ngIf]="darkModeService.isDarkMode" 
+            [ngIfElse]="textToChange">
+                <span class="dark-mode-theme">Përdor pamjen e çelur</span>
+            </ng-template>
+            <ng-template #textToChange>
+                <span class="light-mode-theme">Përdor pamjen e errët</span>
```

Comment:
Capitalize the words, i.e. `Përdor Pamjen e Errët` instead, same for light mode.

Reply:
Sure